packages/core/src/sso/GoogleWorkspaceSsoConnector/index.ts

```diff
@@ -0,0 +1,58 @@
+import { ConnectorError, ConnectorErrorCodes } from '@logto/connector-kit';
+import { type SsoConnector, SsoProviderName } from '@logto/schemas';
+
+import OidcConnector from '../OidcConnector/index.js';
+import { type SingleSignOnFactory } from '../index.js';
+import { type SingleSignOn } from '../types/index.js';
+import { basicOidcConnectorConfigGuard } from '../types/oidc.js';
+
+// Google use static issue endpoint.
+const googleIssuer = 'https://accounts.google.com';
+
+export class GoogleWorkspaceSsoConnector extends OidcConnector implements SingleSignOn {
+  static googleIssuer = googleIssuer;
+
+  constructor(private readonly _data: SsoConnector) {
```

Comment:
nit: Naming as `data` does not conflict with others?

Reply:
Conflict with the getter. But you are right, looks like getter is useless, making the data as public readonly

Reply:
updated